client/vite.config.ts

```diff
@@ -7,7 +7,7 @@
 import react from '@vitejs/plugin-react';
 // import license from 'rollup-plugin-license';
 // import {defineConfig} from 'vitest/config';
-import {defineConfig} from 'vite';
+import { defineConfig } from 'vite';
```

Comment:
This conflicts with our prettier styles. Should be `{defineConfig}`

Reply:
something is wrong with my vscode config? On save runs a the import fixer but not the linter??

Reply:
I changed to include vite config in the linter. Shouldn't have broken anything 😁